addons/web_editor/tests/test_controller.py

```diff
@@ -171,3 +171,37 @@ def test_font_to_img(self):
         # Image is a play button
         img_reference = Image.open(file_open("web_editor/tests/play.png", "rb"))
         self.assertEqual(img, img_reference, "Result image should be the play button")
+
+    def test_update_field_translation(self):
+        """Checks if translated text could be updated"""
+        self.authenticate('admin', 'admin')
+
+        # Setup
+        br_lang=self.env['res.lang']._activate_lang('pt_BR')
+        en_lang=self.env['res.lang']._activate_lang('en_US')
+
+        website = self.env['website'].browse(1)
+        website.language_ids += br_lang
+        website.default_lang_id = br_lang
+        blog_post = self.env['blog.post'].with_context(lang='pt_BR').create({
+            'name':'Test Blog',
+            'content':'Todos os blogs', 
+            'website_id':website.id,
+        })
```

Comment:
Thanks for the test! Since we are mimicking the blog posting, it requires the dependency of `website` and `website_blog`. However, `web_editor` is a framework module that shouldn't include those dependencies.

This test should probably be moved to `odoo/addons/website_blog/tests/test_website_blog_flow.py` Other than that from my local running, it correctly tests the right translation dictionary is used.